alembic/versions/8e7282ae4840_updated_rtp_tables.py

```diff
@@ -0,0 +1,55 @@
+"""updated rtp tables
+
+Revision ID: 8e7282ae4840
+Revises: e66c069eb92b
+Create Date: 2017-08-13 18:06:40.892734+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '8e7282ae4840'
+down_revision = 'e66c069eb92b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('host_status')
+    op.add_column('rtp_process_record', sa.Column('hera_cal_git_hash', sa.String(length=64), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('hera_cal_git_version', sa.String(length=32), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('hera_qm_git_hash', sa.String(length=64), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('hera_qm_git_version', sa.String(length=32), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('pyuvdata_git_hash', sa.String(length=64), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('pyuvdata_git_version', sa.String(length=32), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('rtp_git_hash', sa.String(length=64), nullable=False))
+    op.add_column('rtp_process_record', sa.Column('rtp_git_version', sa.String(length=32), nullable=False))
+    op.drop_column('rtp_process_record', 'git_hash')
+    op.drop_column('rtp_process_record', 'git_version')
```

Comment:
Looking at the schema onsite, the `time` column is a `bigint` type, and there is no `serial` attribute. So I think it's correct.